tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -73,33 +96,72 @@ SELECT CAST(4.56 AS DECIMAL(6, 2)) - CAST(1.23 AS DECIMAL(6, 2)) AS result;
 ----
 3.32
 
+onlyif mysql
+statement error 1068
+SELECT CAST(12345678912345678900.23 AS DECIMAL(76, 2)) - CAST(1.23 AS DECIMAL(6, 2)) AS result
```

Comment:
How about adding a very number, like 1e50?

Reply:
need my pr to parse large number https://github.com/datafuselabs/databend/pull/10326 
I will try to  finish it today.